modules/cert-manager-override-environment-variables.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-customizing-api-fields.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-override-environment-variables_{context}"]
+= Customizing cert-manager Operator API fields by overriding environment variables
+
+You can override the default environment variables for the {cert-manager-operator} by adding a `spec.controllerConfig` section in the `CertManager` resource.
+
+.Prerequisites
+
+* You have access to the {product-title} cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Edit the `CertManager` resource by running the following command:
++
+[source,terminal]
+----
+$ oc edit certmanager cluster
+----
+
+. Add a `spec.controllerConfig` section with the following override arguments:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1alpha1
+kind: CertManager
+metadata:
+  name: cluster
+  ...
+spec:
+  ...
+  controllerConfig: <1>
+    overrideArgs:
+      - 'HTTP_PROXY' <2>
+      - 'HTTPS_PROXY' <3>
+      - 'NO_PROXY' <4>
+----
+<1> Add the `spec.controllerConfig` section.
+<2>
+<3>
+<4>
+
+. Save the file to apply the changes.
+
+.Verification
+
+* Verify that the cert-manager pods are running by entering the following command:
++
+[source,terminal]
+----
+$ oc get pods -n cert-manager
```

Comment:
Can we use a yaml output of the pods instead? So users could verify if their newly added args/env have come up in the pod spec.
```sh
$ oc get pods -n cert-manager -o yaml
```
should be good, IMO.

Reply:
Similar to my the other comment, except for this one you show the environment variables only.
```
$ oc get pods -n cert-manager -o yaml
---
    name: cert-manager-6d48f88495-c88gd
    namespace: cert-manager
---
env:
      - name: POD_NAMESPACE
        valueFrom:
          fieldRef:
            apiVersion: v1
            fieldPath: metadata.namespace
      - name: HTTP_PROXY
---
```